docs/DafnyRef/UserGuide.md

```diff
@@ -1832,7 +1828,7 @@ and what information it produces about the verification process.
   assignment, the property that every variable is eventually assigned a
   value before it is used.  If false (the default) definite assignment
```

Comment:
If `--relax-definite-assignment` is false, it corresponds to legacy mode 2, if true, it corresponds to legacy mode 1. If `--enforce-determinism` is used, which cannot be combined with `--relax-definite-assignment`, then it corresponds to legacy mode 3.

Reply:
I would take the legacy options out of this list, since it's pretty confusion to have duplicates, and the legacy option documentation can still be found under /help, and in the documentation for older version.

Reply:
Done.